app/javascript/app/pages/national-context/historical-emissions/historical-emissions-selectors/historical-emissions-data-selectors.js

```diff
@@ -190,6 +192,7 @@ export const getChartConfig = createSelector(
     if (!data || isEmpty(data) || !metricSelected) return null;
     const tooltip = getTooltipConfig(yColumnOptions);
     const theme = getThemeConfig(yColumnOptions);
+    colorCache = { ...colorCache, ...theme };
```

Comment:
To avoid overwriting `colorCache` assignment should be `colorCache = { ...theme, ...colorCache }`

Reply:
Also `theme` value (on `config` object, on line `211`) should be updated with `colorCache` to properly apply the colors `theme: colorCache`